easybuild/easyconfigs/c/Circuitscape/Circuitscape-5.12.3-Julia-1.9.2.eb

```diff
@@ -403,6 +403,8 @@ exts_list = [
     }),
 ]
 
-sanity_check_commands = ["julia -e 'using Pkg;Pkg.test(\"Circuitscape\")'"]
+sanity_check_commands = [
+    """julia -e 'using Pkg; Pkg.activate("%(installdir)s/environments/v1.9"); Pkg.test("%(name)s")'""",
```

Comment:
same here

Reply:
and this one fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/20106/commits/ac70aacc13cb3bb9ec2b47313c4f9cc351f9f9df